superset/assets/javascripts/explore/components/ChartContainer.jsx

```diff
@@ -145,6 +150,19 @@ class ChartContainer extends React.PureComponent {
     this.props.actions.runQuery(this.props.formData, true);
   }
 
+  updateChartTitle(newTitle) {
+    const params = {};
+    params.slice_name = newTitle;
+    params.action = 'overwrite';
+    params.form_data = this.props.formData;
+    const saveUrl = '/superset/explore/' +
+      `${this.props.datasourceType}/${this.props.datasourceId}/?${serialize(params)}`;
```

Comment:
Maybe we can reuse the getExploreUrl method here? https://github.com/airbnb/superset/blob/master/superset/assets/javascripts/explore/exploreUtils.js#L4

Or maybe it's time soon to remove move save/overwrite out of `/superset/explore/` and into its own POST endpoint.